linelist_investigations/report_sources/aaa_clean_linelist_2019-10-22.Rmd

```diff
@@ -182,6 +182,18 @@ x <- x_raw %>%
   mutate(date_exposure_start = date_contact_confirmed_case1) %>% 
   mutate(date_exposure_end = enddate1_contact_confirmed) %>%
   mutate(date_vacc = vacc_date) %>%
+  mutate(date_hosp1_start = hosp1_start) %>%
+  mutate(date_hosp2_start = hosp2_start) %>%
+  mutate(date_hosp3_start = hosp3_start) %>%
+  mutate(date_hosp4_start = hosp4_start) %>%
+  mutate(date_hosp5_start = hosp5_start) %>%
```

Comment:
Thanks for converting all these hosp_ variables to start with date_.  
Currently I see hosp5_start values are about 10 numeric digits, like this: 1552176000. Will guess_dates correctly identify what these are meant to be?